docs-win/index.md

```diff
@@ -0,0 +1,297 @@
+## Overview
```

Comment:
the site is built using https://www.mkdocs.org/ with the material theme https://squidfunk.github.io/mkdocs-material/

the organisation of the site is specified in mkdocs.yml (in the root dir), this site has no explicit TOC because it just has an index.md which is the default page. You can look at the beqdesigner for an example of a bigger docs site.

I suggest you reorg these pages to go under docs and update the mkdoc.yml to put OS specific installation instructions on separate page or make them subsections of a common page